Snakefile

```diff
@@ -262,7 +248,7 @@ rule ploidetect:
     """Runs Ploidetect"""
     input:
         rules.preseg.output,
-	    check_docker()
+        rules.ploidetect_install.output if "install_ploidetect" in config.keys() and config["install_ploidetect"] else __file__
```

Comment:
How's this instead - an explicit install_ploidetect option (assumed False if not exiting).
FYI - `__file__` is a magic python value for the current script you are in.

Reply:
what about:
`rules.ploidetect_install.output if config["install_ploidetect"] and not workflow.use_singularity`

this ensures that it's not installed when --use-singularity is set as well

Reply:
Great idea.